test/e2e/framework/kubernetes/no-crashes.go

```diff
@@ -14,6 +14,7 @@ type EnsureStableCluster struct {
 	LabelSelector      string
 	PodNamespace       string
 	KubeConfigFilePath string
+	IgnoreRestart      bool
```

Comment:
Can you add a detailed comment explaining why this flag exists, and why you may choose to ignore container restarts?

Reply:
added comment explaining why and how it should be used.